docs/powergate/index.md

```diff
@@ -0,0 +1,151 @@
+---
+hero_img: /images/powergate-hero.png
+---
+
+# Introduction to the Powergate
+
+The Powergate is an API driven solution for deploying multitiered storage across Filecoin and IPFS. Persistent storage on Filecoin allows rich storage configuration for data such as replication factor, miner selection, deal renewal, and repair. Network available storage is configurable and provided through a connected IPFS peer or pinning network.
+
+## Overview
+
+Powergate is a collection of libraries, modules, and configurations that can used independently, and composed together to integrate Filecoin into your application or storage system. The Powergate is designed to manage one or many Filecoin wallet addresses. Each address in Powergate can be independently managed through the [FFS API](#api) (or grouped together into a single _FFS instance_).
+
+Some benefits of using the Powergate include,
+
+- Make data stored on Filecoin available on the IPFS network easily.
+- Handle long-term storage deal management, including automated renew and repair.
+- Make use of network indices to improve miner selection and deal creation.
+- Manage Filecoin wallet addresses for one or many users.
+- Much more!
+
+### Libraries
```

Comment:
There is also the Golang client included in the powergate repo. Worth mentioning.